### PR TITLE
Add more German translations

### PR DESCRIPTION
**Pull request by julienrf**
Authored by @Manfred1958

### CODE REVIEW

File: frontend/messages/de.json [hunk 1]
```diff
@@ -59,6 +59,128 @@
...
+	"helpOverlay": "Derzeit sehen Sie",
+	"helpWindLayer": "Sie sehen auch",
+	"helpBottom1": "Klicken Sie auf einen Ort auf der Karte, um Meteogramme und Sondierungsdiagramme für diesen Standort anzuzeigen.",
+	"helpBottom2": "Klicken Sie jederzeit auf die Hilfeschaltfläche, um eine Erklärung zu dem zu erhalten, was Sie gerade sehen.",
+	"helpFeedback1": "Etwas funktioniert nicht wie erwartet? Bitte senden Sie eine",
+	"helpFeedback2": "Für weitere Fragen senden Sie uns eine",
+	"helpLayerThQ": "Es gibtdas Potenzial für Thermikflüge an, von 0 % (schlechte Thermik oder sehr starker Wind) bis 100 % (starke, hoch reichende Thermik, schwacher Wind). Suchen Sie nach weißen oder blauen Bereichen (die vollständige Farbskala wird unten rechts auf dem Bildschirm angezeigt). Die Qualität der Thermik berücksichtigt die Schichttiefe der konvektiven Grenzschicht, die Bodenerwärmung und die durchschnittliche Windgeschwindigkeit innerhalb der konvektiven Grenzschicht. Eine Dicke konvektive Grenzschicht, eine starke Erwärmung des Bodens und niedrige Windgeschwindigkeiten erhöhen den Wert dieses Indikators.",
```
Comment (Boran) [inline frontend/messages/de.json:105]: gibtdas > gibt das

**Comment by Boran** [comment]
Looks good. Two questions, would it be better to replace 
a) "1500 m" with "1500m" (readability)?
b) "ß" with ss ? (The "ß" is more old fashioned? I'm not a native speaker and dont have a strong opinion)

**Comment by julienrf** [comment]
> a) "1500 m" with "1500m" (readability)?

Personally, I don’t find “1500m” more readable than “1500 m”. Unless this is the standard way to add a unit to a scalar value in German, I would rather keep “1500 m”.

File: frontend/messages/de.json (hunk 1 above)
Comment (julienrf) [reply frontend/messages/de.json:105]:
```suggestion
	"helpLayerThQ": "Es gibt das Potenzial für Thermikflüge an, von 0 % (schlechte Thermik oder sehr starker Wind) bis 100 % (starke, hoch reichende Thermik, schwacher Wind). Suchen Sie nach weißen oder blauen Bereichen (die vollständige Farbskala wird unten rechts auf dem Bildschirm angezeigt). Die Qualität der Thermik berücksichtigt die Schichttiefe der konvektiven Grenzschicht, die Bodenerwärmung und die durchschnittliche Windgeschwindigkeit innerhalb der konvektiven Grenzschicht. Eine Dicke konvektive Grenzschicht, eine starke Erwärmung des Bodens und niedrige Windgeschwindigkeiten erhöhen den Wert dieses Indikators.",
```

**Comment by julienrf** [comment]
Closing in favor of #131, which takes everything from this PR and adds another commit that fixes the directory layout.

**Pull request closed without merging**